challenges.yml

```diff
@@ -2088,3 +2088,217 @@ extending_prop_transition:
+    `value` of type `String`?
+  explanation: |
+    In vue 2, `v-model` translates to `:value` and an `$emit` with the event
+    `input` and the text to be changed! Just remember that in Vue 3, the
+    default prop to be used in `v-model` is `modelValue`!
+
+    > Alternatively, you can also pass `v-model:value` in Vue 3. The `:value` part
+    > acts as a parameter of the `v-model` *"function"* and tells it that
+    > the prop we want to use is called `value`. Just like in Vue 2!
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          ```vue
+          <input
+              :id="id"
+              :name="id"
+              type="text"
+              :value="value"
+              @input="$emit('input', $event.target.value)"
+          >
+          ```
+        b: |
+          ```vue
+          <input
+              :id="id"
+              :name="id"
+              type="text"
+              :value="value"
+              @input="$emit('input', $event)"
+          >
+          ```
+        c: |
+          ```vue
+          <input
+              :id="id"
+              :name="id"
+              type="text"
+              :value="value"
+              @change="$emit('input', $event.target.value)"
+          >
+          ```
+        d: |
+          ```vue
+          <input
+              :id="id"
+              :name="id"
+              type="text"
+              :value="value"
+              @keyup="$emit('input', $event.target.value)"
+          >
+          ```
+      correct: a
+extend_v_bind_object:
+  question: |
+    `v-bind` is an amazing shortcut that lets us assign multiple, or even dynamic
+    props to a component using one single attribute. Given this piece of code,
+    what are the actual props that the child component will receive?
+
+    ```vue
+    <template>
+        <child-component
+            text="Some text prop here"
+            v-bind="{
+                isValid: true,
+                showText: true,
+            }"
+        />
+    </template>
+    ```
+  explanation: |
+    The fact that `v-bind` is so flexible, is a great feature of Vue. You can
+    mix `v-bind` with regular props and get both passed down to the child
+    component.
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          ```
+          {
+              isValid: true,
+              showText: true,
+          }
+          ```
+        b: |
+          ```
+          {
+              text: 'Some text prop here',
+          }
+          ```
+        c: |
+          ```
+          {
+              isValid: true,
+              showText: true,
+              text: 'Some text prop here',
+          }
+          ```
+      correct: c
+
+extending_checkout_submit:
+  question: |
+    In this video, we show a neat trick that we can do when submitting a form
+    in Vue:
+
+    ```vue
+    <form @submit.prevent="onSubmit">
+        .
+        .
+        .
```

Comment:
we can just do `...` on line line.

Btw, for something like this, does the `vue` highlighting work better or the `html`? It's sort of just HTML... except that the `@submit.prevent` is Vue. But it's not a SFC... so I'm not sure which highlights better.

Reply:
I'm thinking html is better